File: 1_Year/CursoJava/media3.java
import java.text.DecimalFormat;
import java.util.Scanner;

public class media3 {
    public static void main(String[] args) {
		Scanner leitor = new Scanner(System.in);
		DecimalFormat df = new DecimalFormat("0.0");
		double Nota1 = leitor.nextDouble()* 2;
		double Nota2 = leitor.nextDouble()* 3;
		double Nota3 = leitor.nextDouble()* 4;
		double Nota4 = leitor.nextDouble();
        double MEDIA = (Nota1 + Nota2 + Nota3 + Nota4) / 10;
		System.out.println("Media: " + df.format(MEDIA));
		

        if (MEDIA >= 7) {
        	System.out.println("Aluno aprovado.");

        } else if (MEDIA >= 5 && MEDIA < 7) {
        	System.out.println("Aluno em exame.");
        	double NExame = leitor.nextDouble();
			System.out.println("Nota do exame: " + df.format(NExame));
        	MEDIA = (MEDIA + NExame) / 2;

        	if (MEDIA >= 5) {
            	System.out.println("Aluno aprovado.");
        	} else {
            	System.out.println("Aluno reprovado.");
        	}
			System.out.println("Media final: " + df.format(MEDIA));
        } else {
        	System.out.println("Aluno reprovado.");
        }
        leitor.close();
    }
}

File: 1_Year/AED l/www.beecrowd.com.br/Lista 4/Media_3.cpp
#include <stdio.h>

int main()
{
    float n[4], exame, media;

    scanf("%f %f %f %f", &n[0], &n[1], &n[2], &n[3]);
    media = ((n[0]*2) + (n[1]*3) + (n[2]*4) + n[3]) / 10;
    printf("Media: %.1f\n", media);

    if (media>=7)
        printf("Aluno aprovado.\n");

    if (media>5.0 && media<7.0)
    {
        printf("Aluno em exame.\n");
        scanf("%f", &exame);
        printf("Nota do exame: %.1f\n", exame);
        
        media= (media + exame) / 2;

        if ((media)<5)
            printf("Aluno reprovado.\n");

        else if (media>=5)
            printf("Aluno aprovado.\n");

    printf("Media final: %.1f\n", media);
        
    } else if (media<5.0) 
        printf("Aluno reprovado.\n");
    
    return 0;
}